thirdeye-ui/src/app/platform/components/notification-provider-v1/notification-display-v1/notification-display-v1.component.tsx

```diff
@@ -65,6 +65,8 @@ export const NotificationDisplayV1: FunctionComponent<NotificationDisplayV1Props
                     <Snackbar
                         open
                         anchorOrigin={{ vertical: "top", horizontal: "center" }}
+                        autoHideDuration={3000}
```

Comment:
What is the behavior if you have a long list and have them timeout? Also, I think having a 10s timeout would be better. Some of the msg are long and 3s is too short. thoughts?

Reply:
+1, even 20s is more reasonable to start with I think.

Reply:
Ok ya I'll do 10s. The timeout is for each alert serially, so if there are 20 alerts, then first one displays and timeouts out, then next displays and so on.

Reply:
thanks for sharing the compounding thing. 10s looks ok to me. We will do a longer term thing when @spham92  is back.